articles/ros2_threat_model.md

```diff
@@ -186,6 +252,10 @@ attack).
     <td>Physical Safety</td>
     <td>The robotic system must not harm its users or environment.</td>
   </tr>
+  <tr>
+    <td>Robot Integrity</td>
+    <td>The robotic system must not damage itself.</td>
```

Comment:
I don't have a strong opinion there, but compared to expanding the scope of `Physical Safety`, do you see cases where those two columns values won't be identical?

Reply:
+1